_includes/setup/ios-setup.md

```diff
@@ -48,12 +48,17 @@ iOS devices.
    $ brew install ideviceinstaller ios-deploy
    $ brew install cocoapods
    ```
-   If any of these commands fails with an error, run `brew doctor` and follow the instructions 
+   If any of these commands fails with an error, run `brew doctor` and follow the instructions
    for resolving the issue.
-   
-1. Follow the steps defined by Apple to configure Xcode with your Apple Developer account and enable deploying with provisioning profiles. To learn how to set up your profile, see [Apple's official documentation](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingProfiles/MaintainingProfiles.html).
-1. Select your Development Team for the Flutter project.
-   1. Open the default Xcode workspace in your project under `ios/Runner.xcworkspace`. 
-   1. Select the `Runner` project in the left navigation panel.
-   1. In the Target Settings page, make sure your Development Team is selected under **General > Signing > Team**.
+
+1. Follow the Xcode signing flow to provision your project:
+    1. Open the default Xcode workspace in your project by running `open ios/Runner.xcworkspace` in a terminal window from  your Flutter project directory.
+    1. In Xcode, select the `Runner` project in the left navigation panel.
+    1. In the `Runner` target settings page, make sure your Development Team is selected under **General > Signing > Team**. When you select a team, Xcode creates and downloads a Development Certificate, registers your device with your account, and creates and downloads a provisioning profile (if needed).
+        * To start your first iOS development project, you may need to sign into Xcode with your Apple ID.
+        ![Xcode account add](/images/setup/xcode-account.png)  
```

Comment:
Seems like we got some pretty big pictures in some other pages too. Added 720w pictures.